odpi.bom

```diff
@@ -137,10 +137,10 @@ bigtop {
       relNotes = 'Apache Hive'
       version { base = '1.2.1-odpi'; pkg = '1.2.1+odpi'; release = 1 }
       tarball { destination = "apache-${name}-${version.base}-src.tar.gz"
-                source      = destination }
-      url     { download_path = "/$name/$name-${version.base}/"
-                site = "${apache.APACHE_MIRROR}/${download_path}"
-                archive = "${apache.APACHE_ARCHIVE}/${download_path}" }
+                source      = ""odpi-branch-1.2.1.zip" }
```

Comment:
you may want to remove an extra " here, but LGTM otherwise -- try it out and merge away!